fr_FR.lang

```diff
@@ -25,138 +25,138 @@ characterDesigner.accessories=Accessoires
 //[Reference] 'Cancel'
 characterDesigner.cancel=Annuler
 //[Reference] 'Eyes'
-characterDesigner.eyes=Yeux :
+characterDesigner.eyes=Yeux
 //[Reference] 'Gender'
-characterDesigner.gender=Sexe :
+characterDesigner.gender=Sexe
 //[Reference] 'Hair'
-characterDesigner.hair=Cheveux :
+characterDesigner.hair=Cheveux
 //[Reference] 'Hat'
-characterDesigner.hat=Chapeau :
-//[Reference] 'Make yourself pretty and stuff'
-characterDesigner.info=Infos :
+characterDesigner.hat=Chapeau
+//[Reference] 'Make yourself pretty and stuff' TODO The translation seems wrong, but it might be because of context
+characterDesigner.info=Infos
```

Comment:
Pretty sure you're right and that the translation is wrong due to not having proper references back then